compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -13,6 +13,11 @@
 #include "mlir/Dialect/MemRef/Transforms/Passes.h"
 #include "mlir/Transforms/Passes.h"
 
+static llvm::cl::opt<bool>
```

Comment:
For followup: these should be added to the GlobalOptions class instead of as loose CL options.

Reply:
(see how the others in here are done)

Reply:
done